lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1304,13 +1305,13 @@ class EditableTextStyle {
 
   /// This information will be used for changing the style of the hidden input
   /// element, which will match it's size to the size of the editable widget.
-  final double fontSize;
+  final double? fontSize;
   final String fontWeight;
-  final String fontFamily;
+  final String? fontFamily;
   final ui.TextAlign textAlign;
   final ui.TextDirection textDirection;
 
-  String get align => textAlignToCssValue(textAlign, textDirection);
+  String? get align => textAlignToCssValue(textAlign, textDirection);
```

Comment:
this never returns null

Reply:
Done.

Reply:
I still have null assert because in the general case `textAlignToCssValue` can return `null`.

Reply:
Hmm, actually this broke a test. Looks like we do return null sometimes. Let's revisit this in a follow-up PR?